src/main/kotlin/com/zepben/evolve/cim/extensions/ZBEX.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.extensions
+
+/**
+ * Indicates that a member or class is a Zepben extension added to the CIM standard.
+ *
+ * This class should be linked into documentation of all our CIM extensions. Once this has been
+ * linked everywhere you will be able to use find usages to see all of our extension attributes.
+ */
+object ZBEX
```

Comment:
**suggestion:** It could be useful to make this an `annotation class` instead of an object. You could then also annotate the properties themselves, and if the annotation class had `@MustBeDocumented` annotation itself it should theoretically show the property is annotated  with it in any generated kdoc. 
Just a suggestion in case you think there is any benefit.

Reply:
I thought of doing this but I have no idea how and didn't want to spend the time figuring it out. If it is simple then I am happy to add it.

Kdoc does not currently support escaping the square brackets. This was just a way of keeping the docs the same with the ZBEX in the square brackets without the IDE reporting warnings about an unknown object.

Reply:
It would literally just be
```kotlin
@MustBeDocumented
annotation class ZBEX
```

That would work as is (for the doc references) but you could also annotate properties / functions / classes / whatever with `@ZBEX` if you wanted to document "in code" too.

Reply:
Do you think it is valuable to have it in the documentation as well as annotated against the field?

![image](https://github.com/zepben/ewb-sdk-jvm/assets/7707606/76534876-ed3c-4b65-a884-cfc2e164bb87)
![image](https://github.com/zepben/ewb-sdk-jvm/assets/7707606/28d37213-d617-4cbe-8356-0d782e5c57f7)
![image](https://github.com/zepben/ewb-sdk-jvm/assets/7707606/00ce06ee-a36c-4021-af39-5bf68cf7a092)

Reply:
IMO I think it's probably useful as you don't have to find the docstring to easily see it's one of our extensions. Would probably be useful for 3rd parties using the SDK (or newbies to the SDK) who are familiar with CIM.

Reply:
As discussed it will be kept in both places. This gives the added benefit of showing up in the class docstring:
![image](https://github.com/zepben/ewb-sdk-jvm/assets/7707606/17cc8e39-281a-4972-bd96-66df4da2eee5)